src/boot/ksdk1.1.0/warp-kl03-ksdk1.1-powermodes.c

```diff
@@ -289,6 +306,16 @@ warpSetLowPowerMode(WarpPowerMode powerMode, uint32_t sleepSeconds)
 
 		case kWarpPowerModeVLLS3:
 		{
+			/* TODO: this can be replaced using the internal RTC */
```

Comment:
Comment style.

Reply:
Even single line comments? There seems to be a lot of examples within the repository where single lines are done in this way. Should all comments be done this way? What about inline comments?

Reply:
Happy to make the change, but I'm wondering if I should also change some of my other single line inline comments which you didn't explicitly comment on